app/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # syntax = docker/dockerfile:1.2
 
-FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.3 as builder
+FROM  --platform=$TARGETPLATFORM ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.3 as builder
```

Comment:
I managed to get things running for arm without changing "scratch" or adding "--platform" there. It anyway only copy things right, so this should maybe be sufficient.

Reply:
do you have an arm device? otherwise I would assume doing it like this still requires qemu.

so what I would suggest is to properly use cross compilation (-x <arch>)